core/src/main/java/org/elasticsearch/index/query/BoolQueryBuilder.java

```diff
@@ -93,6 +149,25 @@ public BoolQueryBuilder should(QueryBuilder queryBuilder) {
     }
 
     /**
+     * Adds a list of queries that <b>should</b> appear in the matching documents. For a boolean query with no
+     * <tt>MUST</tt> clauses one or more <code>SHOULD</code> clauses must match a document
+     * for the BooleanQuery to match.
+     *
+     * @see #minimumNumberShouldMatch(int)
+     */
+    public BoolQueryBuilder should(List<QueryBuilder> queryBuilders) {
+        shouldClauses.addAll(queryBuilders);
+        return this;
+    }
+
+    /**
+     * Gets the queries that <b>should</b> appear in the matching documents.
```

Comment:
can't quite read it. how about "Returns the clauses that should be matched by the returned documents" ? in general should appear is probably not the best way to say it (I'm not a native english speaker though!)